native/wasmex/src/namespace.rs

```diff
@@ -83,18 +83,16 @@ fn create_imported_function(
         .map(|term| term_to_arg_type(term))
         .collect::<Result<Vec<Type>, _>>()?;
 
-    let signature = Arc::new(FuncSig::new(params_signature.clone(), results_signature.clone()));
+    let signature = Arc::new(FuncSig::new(
+        params_signature.clone(),
+        results_signature.clone(),
+    ));
 
     Ok(DynamicFunc::new(
         signature,
         move |_ctx: &mut Ctx, params: &[Value]| -> Vec<runtime::Value> {
             let callback_token = ResourceArc::new(CallbackTokenResource {
-                token: (
-                           Mutex::new(None),
-                           Condvar::new(),
-                           params_signature.clone(),
-                           results_signature.clone(),
-                       ),
+                token: (Mutex::new(None), Condvar::new(), results_signature.clone()),
```

Comment:
Oh I see we only need to know the results here. I wasn't sure and so added more info into it. Sorry about that.

Reply:
Please don't be sorry, this is why we are a team here 🤗